agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -152,18 +151,21 @@ def add_variant(
             click.echo(click.style(f"Error while adding variant: {ex}", fg="red"))
         return None
 
-    # Get user profile
-    user_id = client.retrieve_user_id(host, api_key)
+    parent_directory = Path(__file__).parent.parent
```

Comment:
This is not the right config.toml please see https://github.com/Agenta-AI/agenta_ee/blob/8360bfa28749194b002f14aca37ee652a2bd2e5a/agenta-cli/agenta/cli/helper.py#L14

Reply:
I get this:
![image](https://github.com/Agenta-AI/agenta/assets/55067204/02f8be08-2d3c-40d5-a7e0-0ce19443888f)

Reply:
sorry about that @aybruhm here is the correct one
https://github.com/Agenta-AI/agenta/blob/1243ee54883048e32846f5bd471c9bfda67c2678/agenta-cli/agenta/cli/helper.py#L14